client/app/app.scss

```diff
@@ -58,3 +58,8 @@ body,
     width: 100%;
   }
 }
+
+// Makes the outline on focus less intrusive.
+:focus {
+  outline: thin dotted !important;
+}
```

Comment:
The focus outline is for being able to use the application vis keyboard. If it is removed there is no indication that an element focused making keyboard input impossible. We could make custom style for focus highlighting at some point.

http://www.outlinenone.com/